doc/tutorial/get_started.md

```diff
@@ -267,48 +264,50 @@ Complete the following steps:
    1. Select `yes` to select more than one cluster member.
    1. As the address for MicroCloud's internal traffic, select the listed IPv4 address.
    1. Copy the session passphrase.
-   1. Head to the other servers (`micro2`, `micro3`, and `micro4`) and start the join process:
 
-          lxc exec micro2 microcloud join
+1. Head to the other VMs (`micro2`, `micro3`, and `micro4`) and start the join process on each:
+
+       lxc exec micro2 microcloud join
 
-         ```{tip}
+   ```{tip}
+   Open three additional terminals to run the commands concurrently.
+   ```
 
-         Open up three additional terminals to run the commands concurrently.
-         ```
+   In each joining cluster member, select the address for MicroCloud's internal traffic that was chosen on `micro1`.
```

Comment:
"from the same subnet that was chosen on `micro01`"?

Reply:
Maybe I am wrong but the sentence sounds like you should select the same address for MicroCloud's internal traffic that was already chosen on `micro1`.
But in fact it's a different address in the same subnet.

Reply:
You are totally right. I updated it to use the same wording as on the initiating cluster member:
>In each joining cluster member, select the listed IPv4 address for MicroCloud's internal traffic.

I also added to the commit an updated screengrab of the join process that shows the address options, to help make it more clear. Thanks!

Reply:
Thx!